docs/src/modules/spring/pages/spring-boot-integration.adoc

```diff
@@ -0,0 +1,32 @@
+= Spring Boot Integration
+
+The Kalix Spring SDK provides a familiar developer experience for Java developers that are used to Spring. Its `Main`
+class is annotated with `@SpringBootApplication` and starts exactly the same way as in a Spring Boot application, namely
+by calling `SpringApplication.run(Main.class, args)`.
+
+For all matters, it's a Spring Boot application that runs and connects with the Kalix Proxy. As such, your application can benefit of most features available to Spring applications. However, some features are not available or are not recommended when building a Kalix application based on the Spring SDK. Those are explained in the sections below.
+
+== Dependency Injection
+
+*Actions*, *Entities* and *Views* are all Kalix components, and they get special treatment.
+
+When your application starts, it will scan the classpath for any class extending one of the Kalix components. Each time such a component is found, it's analysed in order to generate the necessary metadata that Kalix requires to be able to interact with them. The component is then registered to Kalix and made available to the Proxy through Kalix internal protocol.
+
+It's important to understand that it's not possible to interact directly with a Kalix component. All interaction must happen through the Proxy. That's how Kalix manages state and distribution. That said, it's not possible to use dependency injection to inject Kalix components inside other components. Doing so will raise a runtime error.
```

Comment:
Some re-wording.. see what you think

```suggestion
It's important to understand that interacting directly with a Kalix component it's NOT possible. All interactions must happen through the Proxy since that's how Kalix manages state and distribution. Therefore, it's not possible to use dependency injection to inject Kalix components inside other components and doing so will raise a runtime error.
```